### DISCUSSION TITLE
plot distance problem

### DISCUSSION THREAD

**Discussion by sujan1746** [ROBelgium/MSNoise#267] [Q&A] [answered]
Hi I plot distances which used windsorizing as 3 and one-bit normalization.

The former one looks well as below.
![image](https://user-images.githubusercontent.com/89066297/153835434-6724a804-e095-404c-b50a-efaaf542e04e.png)

But the latter one looks a little bit too strong (?).
![image](https://user-images.githubusercontent.com/89066297/153809440-254521fa-5a52-4d24-b2e2-d8f698c4c4df.png)

So could you please let me know why this happened?

The data is from 2013-2015 and sampling rate is 200Hz.
The their config.csv set is as below and the only difference is windsorizing 3 and -1.
```
data_folder 	/home/sujan/JJ_h/DATA
	output_folder 	CROSS_CORRELATIONS
	data_structure 	SDS
	archive_format 	
	network 	*
	channels 	*
	startdate 	1970-01-01
	enddate 	2015-12-01
	analysis_duration 	86400
	cc_sampling_rate 	50
	resampling_method 	Lanczos
	preprocess_lowpass 	70
	preprocess_highpass 	0.005
	preprocess_max_gap 	10.0
	preprocess_taper_length 	20.0
	remove_response 	N
	response_format 	dataless
	response_path 	inventory
	response_prefilt 	(0.005, 0.006, 94.0, 99.0)
	maxlag 	120.
	corr_duration 	3600.
	overlap 	0.5
	windsorizing 	-1
	whitening 	A
	whitening_type 	B
	stack_method 	linear
	pws_timegate 	10.0
	pws_power 	2.0
	crondays 	1
	components_to_compute 	ZZ
	cc_type 	CC
	components_to_compute_single_station 	
	cc_type_single_station_AC 	CC
	cc_type_single_station_SC 	CC
	autocorr 	N
	keep_all 	N
	keep_days 	Y
	ref_begin 	1970-01-01
	ref_end 	2021-01-01
	mov_stack 	1,7,14,31,62
	export_format 	MSEED
	sac_format 	doublets
	dtt_lag 	static
	dtt_v 	1.0
	dtt_minlag 	5.0
	dtt_width 	30.0
	dtt_sides 	both
	dtt_mincoh 	0.65
	dtt_maxerr 	0.1
	dtt_maxdt 	0.1
	plugins 	
	hpc 	N
	stretching_max 	0.01
	stretching_nsteps 	1000 
```

**Comment by ThomasLecocq** [comment, marked as answer]
I don't understand what you mean by "strong" ? Your filters are not the same either, so I don't expect the two figures to look the same ?

**Reply by sujan1746** [reply]
Sorry for misuploaded. The previous one is one of the subfrequencies after computing cross correlation in 0.01-15Hz Bandpass filter.

The current plot is the CC results in 0.01-15Hz Bandpass filter.

So the only difference in the plots is windsorizing as 3 and -1.

The "strong" means it looks having more "wiggled", "pointed", and "bolded" results in distance plot of one-bit normalization as below. 
![image](https://user-images.githubusercontent.com/89066297/153836455-cf13c9c3-656b-4aa2-b948-b4046b644707.png)

I mean the former (windsorizing 3) graph looks smoother.

**Reply by ThomasLecocq** [reply]
it's really difficult to see/understand what the problem is/could be. For sure it's linked to both the quality of your input waveforms, **and** the 1-bit normalisation which induces really strange signals in the data (corners, angles) that don't play nice with FFTs. This is why I never use 1-bit.

**Reply by John8974** [reply]
Oh... I saw many references which used 1-bit normalization for removing the effects of earthquakes. So I thought I have to use 1-bit norm for removing earthquakes..
Then is there any way to use 1-bit normalization?

**Reply by ThomasLecocq** [reply]
It's there, you can test it. But 1-bit was mainly advertised by Bensen for having "nice" CCFs, which are "nice" mostly for dispersion/tomography purposes. The MSNoise standard is 3*RMS clipping (winsorization, Tukey), which is also effectively reducing the effect of earthquakes. The main issue with 1-bit come from the fact, maybe, you're using a very low lower frequency bound, but I am just guessing here, it's impossble to say from your screenshots if there are not other issues with the data. But again, as the 3rms looks good (quite nice actually), I'd just go with that !

**Reply by sujan1746** [reply]
Hmm I got it. I would test and compare both of them. Thank you for letting me know about 1-bit norm!
